docs/samd/pinout.rst

```diff
@@ -0,0 +1,681 @@
+.. _samd_pinout:
+
+Pinout for the SAMD machine modules
+===================================
+
+The assignment of device functions to pins is very flexible. The same function may be used
+at different pins. The representation of the assignment choices are given by a table,
+which is a subset of the MCU's Pin MUX table and is specific to each board, as the
+available pin set varies. The structure of the table is the same for each board, but
+the set of rows is different.
+
+.. _samd21_pinout_table:
+
+Adafruit ItsyBitsy M0 Express pin assignment table
+--------------------------------------------------
+
+|
```

Comment:
What do these "|" characters do?  They aren't used elsewhere in the docs (except for mimxrt, which I think you also wrote).

Reply:
But shouldn't you let Sphinx/HTML/latex decide in the spacing and layout?